warehouse/packaging/utils.py

```diff
@@ -97,6 +100,7 @@ def _simple_detail(project, request):
                     if file.metadata_file_sha256_digest
                     else False
                 ),
+                "provenance": integrity_service.get_provenance_digest(file),
```

Comment:
```suggestion
                "provenance_digest": integrity_service.get_provenance_digest(file),
```
Or something similar

Reply:
This key name is defined by the [PEP 740](https://peps.python.org/pep-0740/#json-based-simple-api).
We could slightly tweak the PEP if we want to update the name here, but we should keep both synced.

Reply:
This is just the variable name that's passed to the Jinja template for the /simple page, we can make it whatever we want.

Reply:
I must be missing something here but I think the `_simple_detail` function is called by [`simple_detail`](https://github.com/pypi/warehouse/blob/4271f2e9e7d9ec28aa5a5a531d3b004bb5f1375f/warehouse/api/simple.py#L112-L113) and used when returning the `JSON` response for the `simple` API.

I have tried with :
```
requests.get("http://localhost/simple/requests", headers={"Accept": "application/vnd.pypi.simple.v1+json"}).json()
```

And I effectively get the new key in the answer : 

```
   'provenance_digest': None
```

Reply:
Ah, you're right, forgot it was being re-used there. Nevermind!